rules/windows/defense_evasion_defender_disabled_via_registry.toml

```diff
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2020/12/23"
+maturity = "production"
+updated_date = "2021/02/08"
+
+[rule]
+author = ["Elastic"]
+description = "DefenderControl is a portable utility used to disable Windows Defender through the use of the command-line or a GUI."
```

Comment:
prob dont need to specifically focus on `DefenderControl` now since its a reg mod

Reply:
How's this?
```
description = "Identifies when the Windows Defender service is disabled or set to be started manually."
```
Also, I can probably remove the fp section.

Reply:
Sounds good, I'd add something about the registry modification in there too. I'd agree, could remove FP section. LGTM!

> description = "Identifies modifications to the Windows Defender registry settings to disable the service or set the service to be started manually."